pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/NoopOutputStream.java

```diff
@@ -13,24 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.linkedin.gradle.python.plugin
+package com.linkedin.gradle.python.util;
 
-import com.linkedin.gradle.python.PythonExtension
-import org.gradle.api.Plugin
-import org.gradle.api.Project
+import java.io.OutputStream;
 
-abstract class PythonBasePlugin implements Plugin<Project> {
-
-  public PythonExtension settings
-  public Project project
-
-  @Override
-  void apply(Project target) {
-    this.project = target
-    target.plugins.apply(PythonPlugin)
-    settings = project.getExtensions().getByType(PythonExtension)
-    applyTo(target)
-  }
-
-  abstract void applyTo(Project project)
+public class NoopOutputStream extends OutputStream {
+    @Override
+    public void write(int b) {
+        //noop
```

Comment:
Can you please add one space between the comment signs and the text: `// noop`.
Similarly for other instances of this pattern in this patch.
You can find them by searching for `//n` on the review page in the browser.